modules/007-registrypackages/images/docker-centos/scripts/install

```diff
@@ -14,14 +14,30 @@
 # limitations under the License.
 
 set -Eeo pipefail
-PACKAGES_TO_INSTALL=""
-if ! rpm --quiet -q "$(rpm -qip docker-ce-cli.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="docker-ce-cli.x86_64.rpm"
+packages_to_install=""
+package_docker_ce="docker-ce"
+package_file_docker_ce="${package}.x86_64.rpm"
```

Comment:
same as here https://github.com/deckhouse/deckhouse/pull/1621/files#r881558234

Reply:
yes i is error, thanks